includes/zigbee2mqtt/configuration.yaml

```diff
@@ -0,0 +1,23 @@
+frontend:
+  port: 8081
+homeassistant: false
+permit_join: true
+mqtt:
+  base_topic: zigbee2mqtt
+  server: mqtt://localhost
+  user: %user
+  #%password: 
```

Comment:
Why commented out?

Reply:
It was because the config file did not accept an empty password. If a password is set the line is uncommented.
I know it looks a bit weired. I try to find a better way

Reply:
It worked. No it looks clean and understandable